marimo/_server2/tests/conftest.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2024 Marimo. All rights reserved.
```

Comment:
Once you consolidate `_server2` into the `_server` folder, let's move these tests into `tests/_server`